src/modules/Popup/Popup.js

```diff
@@ -0,0 +1,196 @@
+import React, { Component, PropTypes } from 'react'
+import { findDOMNode } from 'react-dom'
+import { META } from '../../lib'
+import Portal from '../../addons/Portal/Portal'
+import cx from 'classnames'
+import isString from 'lodash/isString'
```

Comment:
Indeed, let's use the direct filename import. I've commented in #524 to reflect this issue.